CMakeLists.txt

```diff
@@ -713,11 +713,14 @@ set_target_properties(zigcpp PROPERTIES
     COMPILE_FLAGS ${EXE_CFLAGS}
 )
 
+find_library(ZLIB NAMES libz.a z zlib libz)
```

Comment:
Why is `libz.a` first? shouldn't the following `z` find it anyway?

Reply:
I copied this line from another place in the `CMakeLists.txt` in order to quickly get this working to test the zig-bootstrap changes. This could probably be cleaned up (I'm no cmake expert) and should probably be put behind some `ZIG_USE_ZLIB` flag which is automatically enabled if `ZIG_STATIC` is true.